avo-hq/avo
# fix: namespaced resources broken

# Description


Fixes an issue that occurred with namespace resources like `Super::Project` would have issues generating and finding routes.

# Checklist:



## Manual review steps


1. Go to the `Super project resource`
2. Make sure all CRUD operations work
3. Do the same for a resource that's not namespaced

Manual reviewer: please leave a comment with output from the test if that's the case.
